imports/plugins/included/product-variant/server/i18n/en.json

```diff
@@ -3,11 +3,24 @@
   "ns": "reaction-product-variant",
   "translation": {
     "reaction-product-variant": {
+      "admin": {
+        "helpText": {
+          "compareAtPrice": "Original price or MSRP",
+          "optionInventoryQuantity": "Inventory",
+          "optionTitle": "Displayed on Product Detail Page",
```

Comment:
Is the Product Detail Page a proper noun in Reaction? Could probably use some docs on what we consider to be proper nouns that should be upper cased @machikoyasuda

Reply:
If it's not officially... I think it should be.